docs/DeveloperGuide.adoc

```diff
@@ -721,17 +721,51 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 |Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
+|`* * *` | new user | see usage instructions | refer to instructions when I forget how to use the App
+|`* * *` | user | add a new person |
+|`* * *` | user | delete a person | remove entries that I no longer need
+|`* * *` | user | find a person by name | locate details of persons without having to go through the entire list
+|`* *` | user | hide [private contact details](#private-contact-detail) by default | minimize chance of someone else seeing them by accident
+|`*` | user with many persons in the address book | sort persons by name | locate a person easily
+|`*` | clumsy user | undo a command | correct a mistake that was previously executed
+|`* *` |user | redo the command | correct the mistake of undoing too many commands
+|`* *` | user | view the history of commands | check what I have done in the past
+|`* * *` | user | edit a person | change details of a person easily
+|`* *` | user | import persons | share with other users and migrate from other systems
+|`* *` | user | export persons | share with other users and migrate from other systems
+|`* * *` | user | tag a person | store some notes about certain people
+|`* * *` | user | categorize persons | easily organize contacts and later on find wrt group
+|`* *` | user with many persons in the address book | apply filter: category, tag, name | easily view and process only certain group
+|`*` | user with many persons in the address book | list categories with statistics | see how my contacts are grouped
+|`* *` | user | use password to get access to my contacts | protect my data
+|`* *` | user | clear the address book | start anew
+|`* *` | user | see suggestions while typing | type commands faster
+|`* *` | impatient user | get the previously entered commands into the input field | type commands faster
+|`* * *` | user with many persons in the address book | find all the persons that satisfy part of the name (eg. Jo*) | search without remembering all the details
+|`* * *` | user | get closest result if my input doesn't match any name | I get results even if I misspell
+|`* * *` | user | add a picture to a person | recall easily who the person is
+|`* *` | new user | use a tutorial | learn the features of the app quickly
+|`* *` | user | remove/categorize/tag all the persons selected by a filter | process multiple contacts quickly
+|`* * *` | user | add a social media link to a person | get more info about a person in one place (in the App)
+|`* *` | impatient user | use hotkeys | navigate faster
+|`* * *` | user | be warned before execution of a hazardous command | prevent from disastrous mistakes
+|`* * *` | user | change the theme of the GUI (dark/bright) | prevent eye strain or simply customize
+|`* *` | user | add new fields to all the persons | customize the App
+|`* * *` | user | validate persons details when adding
```

Comment:
from this place on, because you havent made the last column, it is all broken when displaying